.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "windows-machine-config-bootstrapper"]
 	path = windows-machine-config-bootstrapper
 	url = https://github.com/openshift/windows-machine-config-bootstrapper
-	branch = release-4.8
+	branch = community-4.8
 [submodule "ovn-kubernetes"]
```

Comment:
The release-4.8 branch saw a bump in OVN. Why did that not happen in the community-4.8 branch?

Reply:
@sebsoto ping

Reply:
That was because I used the `-r` option for the script which limits the functionality to update one submodule. Fixed by doing some manual cherry-picking.